### PR TITLE
#60 created  using function isset()

### PR DESCRIPTION
**Pull request by ReginaBorisova**
Working on #60 Closing Chapter 5.
    -modified index.php; 

Cannot find a way how to make other pages work without work.php, about.php, and contact.php in the main directory. When I delete them, a 404 occurs.

Request to review the code and maybe give a hint.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -1,17 +1,6 @@
...
-    $content = 'home';
-    require_once __DIR__ . '/template/page.phtml';
-    ?>
-</main>
-<?php require_once __DIR__ . '/template/footer.phtml'; ?>
-</body>
-</html>
+<?php
+require_once __DIR__ . '/config/variables.php';
+$content = isset($_GET['page']) ? $_GET['page'] : 'index';
```
Comment (sashas777) [inline index.php]: @ReginaBorisova 
this is correct.   in the page.phtml  require one file with the name that is the value of the `$content` variable.

File: template/page.phtml [hunk 2]
```diff
@@ -6,9 +6,9 @@
 </head>
 <body>
   <?php require_once __DIR__.'/../template/menu.phtml';?>
-<main id="<?php echo $content;?>">
-  <?php require_once __DIR__.'/../template/content/'.$content.'.phtml';?>
-</main>
+  <main id="<?php echo $content; ?>">
+      <?php require_once __DIR__ . '/../template/content/' . $content . '.phtml'; ?>
```
Comment (sashas777) [inline template/page.phtml]: @ReginaBorisova 
This seems correct as well. If file is not found check the path to the file

**Review by sashas777 — CHANGES_REQUESTED** [review]

**Comment by sashas777** [comment]
@ReginaBorisova 
I updated the task. There was a typo in the line:
```
https://rborisova.sashas.org/index.php?page=about - opens the same page as https://rborisova.sashas.org/about.php
```

**Comment by ReginaBorisova** [comment]
#60 Closing Chapter 5 
     - Based on my knowledge of arrays, require_once, and $_GET parameter modified the code so every link has a page parameter
     - Deleted files: about.php, contact.php, work.php;
    
After pushing realized that the code is not clean. So just request your review, but I am aware that work on cleaning needs to be done.

File: template/content/about.phtml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+<?php require_once __DIR__ . '/../../config/variables_about.php'; ?>
```
Comment (sashas777) [inline template/content/about.phtml]: @ReginaBorisova 
Please change this code to dynamically load variables required for a page based on a page and have in in the variables file

File: template/menu.phtml [hunk 4]
```diff
@@ -11,7 +11,8 @@
         <ul class="menu-nav">
             <?php foreach ($menuitems as $url => $label): ?>
                 <li class="nav-item">
-                    <a href="<?= $url ?>" class="nav-link" <?= getCurrentPageId($url) ?>"><?= $label ?></a>
+                    <a href="index.php?page=<?= basename($url, '.php') ?>" class="nav-link <?= getCurrentPageId($url) ?>"><?= $label ?></a>
```
Comment (sashas777) [inline template/menu.phtml]: Due to security its better to have parameters as text, not as the filename. So the URL will be index.php?page={pageName} where {pageName} won't have .php extension

File: config/variables.php [hunk 5]
```diff
@@ -13,10 +13,11 @@
...
     'contact.php'=>'Contact Me'
 ];
-$menuitems=[
-    '/index.php'=> 'Home',
-    '/about.php'=> 'About Me',
-    '/work.php' => 'My Work',
-    '/contact.php'=>'How to reach me'
+$menuitems = [
+    'index.php' => 'Home',
+    'about.php' => 'About Me',
```
Comment (sashas777) [inline config/variables.php]: Are these links correct?

**Review by sashas777 — CHANGES_REQUESTED** [review]

File: config/variables.php (hunk 5 above)
Comment (ReginaBorisova) [reply config/variables.php]: Fixed

File: template/menu.phtml (hunk 4 above)
Comment (ReginaBorisova) [reply template/menu.phtml]: Not sure if I understood you right, but did my best

File: template/content/about.phtml (hunk 3 above)
Comment (ReginaBorisova) [reply template/content/about.phtml]: Done

**Comment by ReginaBorisova** [comment]
Resolve #60 Closing Chapter 5. Changes made : 
    - change  code which requires variables_$content.php to dynamically load variables required for a page based on a page and have it in the variables file;
    - Due to security changed parameters as text, not as the filename. Now the URL  index.php?page={pageName} where {pageName} don't have .php extension
    - cleaned unnecessary spaces, and overall cleaned the code.

**Comment by sashas777** [comment]
@ReginaBorisova 
Thanks. there is a conflict. Can you take a look, please?

**Review by sashas777 — APPROVED** [review]

**Pull request merged**